Duplicati/Library/Backend/AliyunOSS/AliyunOSSOptions.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace Duplicati.Library.Backend.AliyunOSS
+{
+    /// <summary>
+    /// Aliyun OSS Configuration
+    /// 阿里云 OSS 配置
```

Comment:
I am a little mixed with having non-english comments. They are likely not being maintained, and we do not have comments in other languages. Does the Chinese comments here help in a way that the English comments do not?

Reply:
ok, I've fixed it.